librad/src/net/gossip.rs

```diff
@@ -101,6 +101,7 @@ where
     Addr: Clone + PartialEq + Eq + Hash,
 {
     Has(Has<Addr, A>),
+    Applied(A),
```

Comment:
Does this belong here? Shouldn't this only be for messages that are actually gossiped between peers?

Reply:
I'm not sure I understand the question, could you elaborate? :)

Reply:
Should the `Applied` variant be here, since it won't be broadcast to peers?

Reply:
`Info` isn't for broadcasting to peers. It's used for announcing protocol events, afaict.

Reply:
Where protocol events are internal for the current peer